session_items.py

```diff
@@ -1,20 +1,39 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    {'id': 1, 'status': 'Not Started', 'title': 'List saved todo items'},
-    {'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added'}
-]
-
+# from flask import session
+import requests
+import logging
+import os
+import json
+
+logging.basicConfig(level=logging.DEBUG)
+boardId = os.environ.get('boardId')
+key = os.environ.get('key')
+token = os.environ.get('token')
+
+# I understand that this is not the best wat to build urls for requests but using the built in functions 
+# and passing in dictionaries for parameters and auth was return a 401 for every request. After several 
+# hours of looking through docs and I gave up and reverted back to this... If I get time I'll try to get
+# this working properly.
+base_request_url = 'https://api.trello.com/1/boards/'+boardId+'/'
+request_credentials = '?key='+key+'&token='+token
```

Comment:
Minor point is that you can write this slightly more neatly e.g. `f"?key={key}&token={token}` (assuming you're on a recent version of python)